### ISSUE TITLE
Menu not showing up

### ISSUE THREAD

**Issue by calhix** [riscygeek/billig-sweeper#9]
Loving the new updates! It seems though that M is not bringing up the menu anymore (on pinephone at least).

**Comment by riscygeek** [comment]
That's odd. On my laptop it works. Does ESC open the menu though?

**Comment by calhix** [comment]
I can't figure out where ESC on the pp keyboard is haha.

**Comment by calhix** [comment]
Don't think the key is there.

**Comment by riscygeek** [comment]
Which DE do you use?

**Comment by calhix** [comment]
Mobian phosh

**Comment by riscygeek** [comment]
Okay, can't really help. I use Plasma Mobile on Manjaro.

**Comment by calhix** [comment]
Ah OK, does M bring up the menu there?

**Comment by riscygeek** [comment]
I can't bring up the keyboard.
Also the zoom and right-click don't work, do you have the same issue?

**Comment by calhix** [comment]
You may want to just install a fresh image if you haven't used it lately. Yeah zoom and right click (or long press) doesn't work.

**Comment by riscygeek** [comment]
Yeah, I haven't used my PP for a few months. I'm gonna reflash it.

I'm also gonna have to look at how touchscreens work in SDL2 (graphics library).

Btw, off topic, do you know a good Signal App for the Pinephone?
I tried Axolotl back then but it didn't work out for me.

**Comment by calhix** [comment]
Sounds good and yeah that's the main issue I have as well, I also have been using Flare and it may be better but sadly both don't have notifications yet.

**Comment by riscygeek** [comment]
Btw do you use an Arch-based distro? If yes, I'm currently in the process of making a PKGBUILD for the AUR.
That would make it easier to test the project.

**Comment by calhix** [comment]
Using mobian currently which is based on Debian but I think I have manjaro on my SD card so I can test it there, may just need to reflash it since I haven't used it for a few months either.

**Comment by calhix** [comment]
Hmm, so sometimes all keybinds work just fine actually but it seems that none work when I lock the screen and unlock again while the app is open, not sure if that was the case before. Also sometimes just none of the keybinds work but if I restart the app they will work, sometimes I need to restart the app a few times.

**Comment by riscygeek** [comment]
>  If so maybe you could add a small menu button?
I'd like to create a touchscreen gesture (if possible) instead. I hope that would be okay too.

**Comment by calhix** [comment]
For sure that works.

**Comment by riscygeek** [comment]
> Hmm, so sometimes all keybinds work just fine actually but it seems that none work when I lock the screen and unlock again while the app is open, not sure if that was the case before. Also sometimes just none of the keybinds work but if I restart the app they will work, sometimes I need to restart the app a few times, perhaps Phosh related? If so maybe you could add a small menu button?

I can confirm that I have the same issue on Mobian Phosh. After starting the App and going to the App Launcher and back keybindings don't work. On the other side ESC and F1 don't work at all on Phosh.

**Comment by calhix** [comment]
Sounds about right. With the panning ability now, you could have a taskbar sort of thing at the top of the screen with a timer and menu button or something and it wouldn't take up much space if you think it would work alright. Your app is very loosely similar to antimine on android with how the field and panning works so its something to think about.
